admin/users/views.py

```diff
@@ -63,8 +63,8 @@ def delete(self, request, *args, **kwargs):
                 if 'ham_confirmed' in user.system_tags:
                     user.tags.through.objects.filter(tag__name='ham_confirmed').delete()
```

Comment:
Whenever an old system tag is deleted as a result of a spam marking action, I left it in place, as kind of a middle ground between getting rid of all existing tags and moving over entirely to the new system. What do you think of this?

Reply:
It's a good failsafe, I wish there was some documentation about what the systems tags mean, their history, etc, though I don't know where that would go. Too bad we can't just tag the tags.